roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/configure-worker-node.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Include node network
+  ansible.builtin.include_tasks: node-network.yml
+  with_items: "{{ __sap_hypervisor_node_preconfigure_register_worker.networks }}"
+  loop_control:
+    loop_var: __sap_hypervisor_node_preconfigure_register_worker_network
+    index_var: __sap_hypervisor_node_preconfigure_register_worker_network_nr
+  when: __sap_hypervisor_node_preconfigure_register_worker.networks is defined
+
+- name: "Create MCP for {{ __sap_hypervisor_node_preconfigure_register_worker.name }}"
+  kubernetes.core.k8s:
+    template:
+      path: "mcp.yml.j2"
+    state: present
+
+- name: Pause so cluster can process config
```

Comment:
It should be possible to wait for all MCP / MC to be applied and nodes to be ready.